app/src/main/java/com/example/appmarvel/service/utils/extension/ImageResponseGetUrl.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.appmarvel.service.utils.extension
+
+import com.example.appmarvel.service.response.ImageResponse
+import com.example.appmarvel.service.utils.Constants.DOT
+
+fun ImageResponse.getURL(): String {
+    return "${this.path}${DOT}${this.extension}"
```

Comment:
the curly braces around the DOT constant are not necessary

Reply:
Done